engine/src/main/java/org/openobservatory/engine/OONICheckInConfig.java

```diff
@@ -0,0 +1,73 @@
+package org.openobservatory.engine;
```

Comment:
I thought we wanted to remove the `engine` wrappers?

Reply:
Not now, I just added more classes to the engine that is another task https://github.com/ooni/probe/issues/1366